TasksFromYurijL/src/SimpleTasks/Fibanachi.java

```diff
@@ -0,0 +1,77 @@
+package SimpleTasks;

```

Comment:
please name packages according to naming conventions http://www.oracle.com/technetwork/java/codeconventions-135099.html

Reply:
Mast I change the name of the package in current progect? I dont know haw to do it?